charts/opentelemetry-collector/values.yaml

```diff
@@ -80,6 +80,8 @@ presets:
 configMap:
   # Specifies whether a configMap should be created (true by default)
   create: true
+  # Specifies whether the configMap should be mounted by chart name (false by default) 
+  mount: false
```

Comment:
Actually this will break users who are setting `create` to false and expecting nothing to mount :/ So `create` does need to stay in the `if` statement or we have to treat this as a breaking change.

Reply:
Okay keeping it the same